common/app/services/S3.scala

```diff
@@ -34,6 +34,7 @@ trait S3 extends Logging {
 
       val request = new GetObjectRequest(bucket, key)
       val result = client.getObject(request)
+      log.info(s"S3 got ${result.getObjectMetadata.getContentLength} bytes from ${result.getKey}")
 
```

Comment:
we could but it would be more tricky, and the problem we're investigating is too much bandwidth not too long time.  the error case is probably logged elsewhere, Idon't think we're seeing errors